.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Please remove this file.

Also make sure to fixup the style by running `make style-fix` in the top-level directory prior to committing.